Parsers/ASimFileEvent/Parsers/vimFileEventSentinelOne.yaml

```diff
@@ -0,0 +1,109 @@
+Parser:
+  Title: File Event Parser for SentinelOne
+  Version: '0.1.1'
+  LastUpdated: July 24, 2023
+Product:
+  Name: SentinelOne
+Normalization:
+  Schema: FileEvent
+  Version: '0.2.1'
+References:
+- Title: ASIM File Event Schema
+  Link: https://aka.ms/ASimFileEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: SentinelOne Documentation
+- Link: https://<SOneInstanceDomain>.sentinelone.net/api-doc/overview
```

Comment:
please correct this url

Reply:
As discussed the SentinelOne doesn't have the public documentation available for API.
User can refer from their instance URL.